src/main/java/com/company/model/Satellite.java

```diff
@@ -0,0 +1,64 @@
+package com.company.model;
+
+public class Satellite {
```

Comment:
It would be helpful to make use of Lombok library, instead of manually writing all getter and setters. When you use that library, you can easily create constructor just using one of the @NoArgsConstructor, @RequiredArgsConstructor, @AllArgsConstructor annotations. 

Ref: https://projectlombok.org/features/constructor